docs/Tutorial_VALMOD.ipynb

```diff
@@ -0,0 +1,1344 @@
+{
```

Comment:
<p>Why not perform/declare this substitution above when you mention the substitution for <code>\sigma^prime</code> and <code>\mu^prime</code> ? I feel like this can be consolidated a bit nicer and more consistently?  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/586/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>

Reply:
<p>I see. So, if I understand correctly, you mean.... put eq(3) right after the "note" above. Correct? And then, I can provide equations in this order: (1), (4), (2). </p>

Reply:
<p>Yes, something like that</p>